CMakeLists.txt

```diff
@@ -60,7 +63,7 @@ target_link_libraries( ${LRS_TARGET} PUBLIC rsutils )
 
 if(BUILD_WITH_DDS)
     if (CMAKE_SYSTEM MATCHES "Windows" OR CMAKE_SYSTEM MATCHES "Linux")
-
+        
```

Comment:
This extra line looks redundant. Should not have been approved in original PR.
Will not reject this PR because of this :-)